changelogs/fragments/1681-add_passwordstore_yaml_support.yaml

```diff
@@ -0,0 +1,3 @@
+breaking_changes:
+  - "passwordstore lookup plugin - now parsing a password store entry as YAML if possible, skipping the first line (which by convention only contains the password and nothing else). If it cannot be parsed as YAML, the old ``key: value`` parser will be used to process the entry (https://github.com/ansible-collections/community.general/issues/1673)"
+  - "Can break backwards compatibility if YAML formatted code was parsed in a non-YAML interpreted way, e.g. `foo: [bar, baz]` will become a list with two elements in the new version, but a string '[bar, baz]' in the old."
```

Comment:
Please do this in one fragment item. Otherwise the changelog is unreadable for users, since they have no idea what this fragment is about.